chain/chunks/src/lib.rs

```diff
@@ -260,6 +260,7 @@ impl SealsManager {
                 };
 
                 let chosen = Self::get_random_part_ords(candidates);
+                // `sent` is not accessed, so we don't use the time proxy.
```

Comment:
It is not used right now, but will be enabled after #3657 is merged

Reply:
will fix this catch